Dockerfile

```diff
@@ -0,0 +1,48 @@
+### Base image...
+FROM python:3-slim-buster as Base
+
+# Install poetry...
+RUN pip install poetry
+
+# Copy application code...
+COPY . .
+
+# Install prerequisites
+RUN poetry install --no-dev --no-root
+
+# Expose port to listen on...
+EXPOSE 8000
+
+
+
+### Production image...
+FROM base as production
+
+# Install gunicorn...
+RUN pip install gunicorn
```

Comment:
You should find that this line (and the similar flask one) is unnecessary since your `poetry install ...` stage installs flask & gunicorn anyway as they're registered in your `pyproject.toml` file